packages/component-library/draft/Kaizen/SplitButton/styles.scss

```diff
@@ -159,6 +218,23 @@ $button-width: 124px;
   }
 }
 
+.dropdownButtonDefault {
+  composes: buttonReset buttonDefault;
+}
+
+.dropdownButtonPrimary {
+  composes: buttonReset buttonPrimary;
+
+  // Making the left | right border white to separate the button with dropdown button
+  border-left-color: $kz-color-white !important;
+
+  [dir="rtl"] & {
+    // equal to add-tint($ca-palette-seedling, 10%). Temporary until we upgrade to Zen (Wisteria)
```

Comment:
added this comment that is the same that Michael added in [this](https://github.com/cultureamp/kaizen-design-system/commit/1bc36311c7ba674f8525fbe7b15e2c705caeca8b) commit  in [this](https://github.com/cultureamp/kaizen-design-system/commit/1bc36311c7ba674f8525fbe7b15e2c705caeca8b#diff-5e77baee7dee0b91c4ef99b35bbc011b) file